Sources/Player/Common/Event/NotificationsHandler.swift

```diff
@@ -26,6 +26,8 @@ final class NotificationsHandler {
 			queue.async {
 				self.listener.mediaTransitioned(to: mediaProduct, with: playbackContext)
 			}
+		} else {
+			PlayerWorld.logger?.log(loggable: PlayerLoggable.mediaTransitionedToPlayerItemWithoutRequiredData)
```

Comment:
I think this is going to create a lot of logs. If I remember correctly we always call this twice, the first time the metadata is still not there so we don't notify the SDK consumers.

Reply:
ok, I wanted to do some tracking of the flow of the issue and figured here could indicate when issues in the transition of items. I'll remove it then, and after investigation, see if this would still would be desired for investigation.

Reply:
✅ removed